cli/src/commands/branch/move.rs

```diff
@@ -131,5 +124,13 @@ pub fn cmd_branch_move(
         ),
     )?;
 
+    if let Some(mut formatter) = ui.status_formatter() {
+        write!(formatter, "Moved {} branches to ", matched_branches.len())?;
+        workspace_command.write_commit_summary(formatter.as_mut(), &target_commit)?;
+        writeln!(formatter)?;
+    }
+    if matched_branches.len() > 1 && args.names.is_empty() {
+        writeln!(ui.hint_default(), "Specify branch by name to update one.")?;
```

Comment:
I think this will be a bit clearer if it says "... update just one of the branches" or "... update just one".

Aside: I haven't reviewed this very carefully, so I'm not officially approving, but it's probably fine. I'm happy to try `jj` with this and complain if something doesn't feel right.

Reply:
Thanks. Updated the hint.